doc/man1/openssl-mac.pod

```diff
@@ -89,7 +89,7 @@ printable characters only).
 
 Used by GMAC to specify an IV in hexadecimal form (two hex digits per byte).
 
-=item B<outlen:int>
+=item B<size:size_t>
```

Comment:
Why would someone reading the manual for a command care that the number is a `size_t`?

Reply:
changed back to int